test/unittests/libdevcrypto/LibSnark.cpp

```diff
@@ -338,6 +334,155 @@ BOOST_AUTO_TEST_CASE(pairingNullInput)
 	BOOST_CHECK(!r.first);
 }
 
+BOOST_AUTO_TEST_CASE(generateRandomPoints)
```

Comment:
Oh damn, @gumb0   Git pulled irrelevant transactions into this PR when I pushed my changes.  I don't know whether it will ignore them as redundant when the final merge comes.
@chriseth do you know?
Worst case - when we are ready to merge I start a new PR.